.well-known/security.txt

```diff
@@ -0,0 +1,6 @@
+# General Issues
+Contact: https://github.com/documenso/documenso/issues/new?assignees=&labels=bug&projects=&template=bug-report.yml
+# Report critical issues privately, to let us take appropriate action before publishing
```

Comment:
This should be updated to "Report critical issues privately to let us take appropriate action before publishing.", I believe.

Also, for better readability, maybe add space before this paragraph? Just a suggestion

Reply:
done. feel free to suggest edit like these directly, i would just have added them